pointerevents/pointerevent_support.js

```diff
@@ -12,10 +12,12 @@ var All_Pointer_Events = [
 
 // Check for conformance to PointerEvent interface
 // TA: 1.1, 1.2, 1.6, 1.7, 1.8, 1.9, 1.10, 1.11, 1.12, 1.13
-function check_PointerEvent(event) {
-    var pointerTestName = event.pointerType + ' ' + event.type;
+function check_PointerEvent(event, testNamePrefix, pointerEventInterface) {
+    if (testNamePrefix === undefined)
+        testNamePrefix = "";
+    var pointerTestName = testNamePrefix + ' ' + event.pointerType + ' ' + event.type;
     test(function () {
-        assert_true(event instanceof PointerEvent, "event is a PointerEvent event");
+        assert_true(event instanceof (pointerEventInterface || PointerEvent), "event is a PointerEvent event");
```

Comment:
|pointerEventInterface| is either null (when checking the event that was fired in the outer frame) which "(pointerEventInterface || PointerEvent)" will be "PointerEvent" or it is the pointer event interface inside the inner frame which then the result of "(pointerEventInterface || PointerEvent)" will be pointerEventInterface.